runtime/doc/dev_vimpatch.txt

```diff
@@ -0,0 +1,303 @@
+  supported by Nvim.
+    - NA files: `src/Make_*`, `src/testdir/Make__*`
+- `if_*.c` changes: `if_python.c` et. al. were removed.
+- `term.c` changes: the Nvim TUI uses `libtermkey` to read terminal sequences;
+  Vim's `term.c` was removed.
+- `job` patches: incompatible API and implementation
+    - NA files: `src/channel_*`, `src/job_*`, `src/testdir/test_channel_*`,
+      `src/testdir/test_job_*`
+- `:terminal` patches that modify NA files: incompatible API and
+  implementation
+    - NA files: `src/terminal_*`, `src/testdir/test_terminal_*`
+- `defaults.vim` patches
+- Most GUI-related changes: Nvim GUIs are implemented external to the core C
+  codebase.
+    - NA files: `src/gui_*`, `src/gvim_*`, `src/GvimExt/*`, `src/testdir/test_gui*`
+- `balloon` changes: Nvim does not support balloon feature
+    - NA files: `src/beval_*`, `src/testdir/test_balloon_*`
+- libvterm changes: Nvim does not vendor libvterm in `src/`.
+- Screendump tests from `test_popupwin.vim`, `test_popupwin_textprop.vim`:
+  https://github.com/neovim/neovim/pull/12741#issuecomment-704677141
+- json changes: incompatible API https://github.com/neovim/neovim/pull/4131
+    - NA files: `src/json*`, `src/testdir/test_json.vim`
+- `test_restricted.vim` restricted mode is removed in
+  https://github.com/neovim/neovim/pull/11996
+- Many tests in `test_prompt_buffer.vim` require incompatible Vim features
+  such as `channel`; they should still be included, but skipped
+- non-runtime documentation: Moved to https://neovim.io/doc/,
+  https://github.com/neovim/neovim/wiki#developers
+    - NA files: `Filelist`, `README`, `INSTALL`,
+- Anything else might be relevant; err on the side of caution, and post an
+  issue if you aren't sure.
+
+==============================================================================
+VERSION.C                                          *dev-vimpatch-version.c*
+
+The list of Vim patches in `src/nvim/version.c` is automatically updated
+https://github.com/neovim/neovim/pull/7780 based on the presence of
+`vim-patch:xxx` tokens in the Nvim git log.
+
+- Don't update `src/nvim/version.c` yourself.
+    - `scripts/vim-patch.sh -p` intentionally omits `version.c` to avoid merge
+      conflicts and save time when porting a patch.
+- The automation script (`scripts/vimpatch.lua`) only recognizes tokens like
+  `vim-patch:8.0.1206`, not `vim-patch:<hash>`.
+
+==============================================================================
+CODE DIFFERENCES                            *dev-vimpatch-code-differences*
+
+The following functions have been removed or deprecated in favor of newer
+alternatives. See `memory.c`
+https://github.com/neovim/neovim/blob/master/src/nvim/memory.c for more
+information.
+
+  -----------------------------------------------------------------------
+  Deprecated or removed                                 Replacement
+  -----------------------------------------------------------------------
```

Comment:
Hoping all of our help pages can use the "flow" layout instead of fixed-width layout. The main tradeoff is that tables like this must be put in codeblocks, until the vimdoc parser can parse content following `column_heading` in a "codeblock" sort of way.

Reply:
bruh I didn't graduate from vimdoc academy, what's a flow layout?

Reply:
Derp, flow layout in CSS for the website. Lol, context pls.